File: app/src/main/java/igrek/songbook/service/transpose/StringWithDelimiter.java
package igrek.songbook.service.transpose;

class StringWithDelimiter {
	
	String str;
	String delimiter;
	
	StringWithDelimiter(String str, String delimiter) {
		this.str = str;
		this.delimiter = delimiter;
	}
	
	StringWithDelimiter(String str) {
		this.str = str;
		this.delimiter = "";
	}
}

File: app/src/main/java/igrek/songbook/chords/transpose/StringWithDelimiter.kt
package igrek.songbook.chords.transpose

internal class StringWithDelimiter {

    var str: String
    var delimiter: String

    constructor(str: String, delimiter: String) {
        this.str = str
        this.delimiter = delimiter
    }

    constructor(str: String) {
        this.str = str
        this.delimiter = ""
    }
}
